src/fadec/src/common.h

```diff
@@ -52,8 +52,8 @@ class EngineRatios {
     return t2;
   }
 
-  FLOAT64 delta2(double mach, double ambientTemp) {
-    double d2 = this->delta(ambientTemp) * pow((1 + 0.2 * powFBW(mach, 2)), 3.5);
+  FLOAT64 delta2(double mach, double ambientPressure) {
+    double d2 = this->delta(ambientPressure) * pow((1 + 0.2 * powFBW(mach, 2)), 3.5);
```

Comment:
Which pow should get the power? I think the standard `pow` works again in the sim but I guess it would be best just to stay consistent :)

Reply:
For performance, powFBW is faster than pow. The catch is that powFBW can only be used with whole number exponents, reason why for a power of 3.5 I still had to use the standard pow funtion.